app/code/Magento/InventoryImportExport/Plugin/Import/SourceItemImporter.php

```diff
@@ -5,16 +5,16 @@
  */
 declare(strict_types=1);
 
-namespace Magento\InventoryImportExport\Model;
+namespace Magento\InventoryImportExport\Plugin\Import;
 
+use Magento\CatalogImportExport\Model\StockItemImporter;
```

Comment:
plugin should know nothing about implementation (Magento\CatalogImportExport\Model\StockItemImporter), just about interface